homework8/task2.py

```diff
@@ -0,0 +1,63 @@
+import sqlite3
+
+
+class TableData:
+    """This is a wrapper class TableData for database table, that when
+    initialized with database name and table acts as collection object
+
+    :param database_name: full path to a database_file
+    :type database_name: str
+    :param table_name: a name of a chosen table inside database_file
+    :type table_name: str
+    """
+
+    def __init__(self, database_name, table_name):
+        """Constructor method"""
+        self.database_name = database_name
+        self.table_name = table_name
+        self.connection = sqlite3.connect(self.database_name)
+        self.connection.row_factory = sqlite3.Row
```

Comment:
It'd be more readable to return dict instead of Row object: https://docs.python.org/3/library/sqlite3.html#sqlite3.Connection.row_factory